Feature/storybook (#40)
infinum/eightshift-frontend-libs: blocks/init/components/scroll-to-top/docs/readme.md

# Scroll To Top Component

> Component used in multiple blocks with all or partial features.

Reviewer: ```suggestion
A component used in multiple blocks with all or partial features.
```

## Dependencies

None

## Implementation
1. Copy/Paste component folder in your project
2. Rename `eightshift-boilerplate` namespace to your project
3. Add or remove features you are going to use.
4. Implement project specific styles.

> 5. Implement in your block/s by providing necesery attributes.

Reviewer: ```suggestion
5. Implement in your block/s by providing necessary attributes.
```

## Usage

> This component can be used on any html elements by simply providing html class defined in `assets/index.js` file.

Reviewer: ```suggestion
This component can be used on any HTML element by simply providing an HTML class defined in `assets/index.js` file.
```
